### DISCUSSION TITLE
How do you turn off `unplugin-vue-components`?

### DISCUSSION THREAD

**Discussion by dhruvkb** [ElMassimo/iles#170] [Q&A]
In my project I would much prefer not using `unplugin-vue-components` and import components manually in every file where needed. Is there a way to turn off this plugin?

The docs [only briefly mention it](https://iles.pages.dev/guide/plugins#unplugin-vue-components) with other Vite plugins but there are no examples on how one can turn these plugins off if not desired.

**Comment by ElMassimo** [comment]
The plugin is used to automatically resolve some internal îles components, like `Island`, `Head`, and your layouts, so it can't be completely turned off.

You could pass `components: { dirs: []` }` in `iles.config.ts` to avoid detecting your own components, but I haven't tested it.
